README.md

```diff
@@ -55,13 +55,15 @@ $ bower install stampit
 or
 $ bower install stampit=https://npmcdn.com/stampit@3/dist/stampit.umd.min.js
 or
-$ bower install stampit=https://unpkg.com/stampit@3.1.2/dist/stampit.umd.js
+$ bower install stampit=https://unpkg.com/stampit@3.1.3/dist/stampit.umd.js
```

Comment:
Noticed this was published as 3.2.0 on NPM but docs here refering to 3.1.3. Probably not a big deal however.

Reply:
Are we still maintaining the Bower version? Bower itself is no longer maintained.

Reply:
Thanks. Will fix.